src/UI/Components/Diagram/stencil/stencil.ts

```diff
@@ -163,18 +167,18 @@ export class StencilSidebar {
     serviceInventories: Inventories,
     service: ServiceModel,
   ) {
-    const instanceTab = new InstanceStencilTab(
+    this.instanceTab = new InstanceStencilTab(
```

Comment:
What lead you to need `this` ?

Reply:
Added `remove()` function for the cleanup to the useEffect,  and to do so I had to have the reference outside from constructor